### PR TITLE
scripts: get_maintainer: Use yaml.safe_load()

### PR DESCRIPTION
**Pull request by mfischer**
Using safe_load() is preferred where possible.

Signed-off-by: Moritz Fischer <moritzf@google.com>

### CODE REVIEW

File: scripts/get_maintainer.py [hunk 1]
```diff
@@ -31,13 +31,7 @@
 import subprocess
 import sys
 
-from yaml import load, YAMLError
-try:
-    # Use the speedier C LibYAML parser if available
-    from yaml import CLoader as Loader
-except ImportError:
-    from yaml import Loader
```
Comment (carlescufi) [inline scripts/get_maintainer.py:39]: Why did you remove this part?

Comment (mfischer) [reply scripts/get_maintainer.py:39]: `safe_load` does not support specifying a loader afaik, the `load()` was the only use case of those imports

Comment (mfischer) [reply scripts/get_maintainer.py]: Actually replaced it with SafeLoader.

**Review by carlescufi — APPROVED** [review]

**Review by mbolivar-nordic — APPROVED** [review]
